### ISSUE TITLE
What is the parameter for ASR model?

### ISSUE THREAD

**Issue by Skywalker-Harrison** [tzyll/ChineseHP#1]
Hello, you mentioned you used Belle-distilwhisper for the task, may I ask the parameter for asr, including temperature, top p and top k in your article?

**Comment by tzyll** [comment]
> Hello, you mentioned you used Belle-distilwhisper for the task, may I ask the parameter for asr, including temperature, top p and top k in your article?

Hi, we used WhisperForConditionalGeneration from transformers. During generate(), just num_beams and num_return_sequences were considered, others default.

**Comment by Skywalker-Harrison** [comment]
Oh, thank you, did you set num_beams and num_return_sequences as 10?

**Comment by tzyll** [comment]
> Oh, thank you, did you set num_beams and num_return_sequences as 10?

Yes, model.generate(input_features, num_beams=n_beam, num_return_sequences=n_beam) where n_beam=10.
